tests/test_popularity_recommender.py

```diff
@@ -16,41 +16,71 @@ def test_calculate_artist_avg_popularity() -> None:
                 "artists",
                 "year",
                 "popularity",
+                "explicit",
+                "acousticness",
+                "danceability",
+                "instrumentalness",
+                "energy",
```

Comment:
Thanks for updating the test file, since the upstream parse_data changed due to new attribute recommender script. We need more att.